src/site/layouts/health_care_region_page.drupal.liquid

```diff
@@ -138,73 +140,108 @@ Example data:
-        <div class="usa-grid usa-grid-full">
-          <div class="usa-width-one-third">
-            Make an appointment
+        <img src="{{ image.derivative.url }}" alt="{{ image.alt }}" title="{{ image.title }}" width="100%">
+        <div class="usa-grid usa-grid-full vads-u-margin-top--1p5">
+          <div class="usa-width-one-third facility-main-cta-item">
+            <a href="{{ entityUrl.path }}/appointments">Make an appointment</a>
           </div>
-          <div class="usa-width-one-third">
-            View A-Z Health Services
+          <div class="usa-width-one-third facility-main-cta-item">
+            <a href="{{ entityUrl.path }}/">View A-Z Health Services</a>
           </div>
-          <div class="usa-width-one-third">
-            Become a patient
+          <div class="usa-width-one-third facility-main-cta-item">
+            <a href="{{ entityUrl.path }}/become-a-patient">Become a patient</a>
           </div>
         </div>
-      </article>
-      <div class="usa-font-lead">
-        {{ fieldIntroText }}
-      </div>
-      <h2>Main Locations</h2>
 
-      <a href="/drupal{{ entityUrl.path }}/locations">See all locations</a>
-
-      <!-- "Manage your health online" section -->
-      <h2>Manage your health online</h2>
-      <div class="usa-grid usa-grid-full">
-        <div class="usa-width-one-half">
-          Refill and track your prescriptions
-        </div>
-        <div class="usa-width-one-half">
-          Get you VA medical records (Blue Button)
-        </div>
-      </div>
-      <div class="usa-grid usa-grid-full">
-        <div class="usa-width-one-half">
-          Send a secure message to your health care team
-        </div>
-        <div class="usa-width-one-half">
-          View your lab and test results
+        <div class="usa-font-lead">
+          <p>{{ fieldIntroText }}</p>
         </div>
-      </div>
-      <div class="usa-grid usa-grid-full">
-        <div class="usa-width-one-half">
-          Schedule and view your appointments
-        </div>
-        <div class="usa-width-one-half">
-          Order hearing aid batteries and prosthetic socks
+
+        <section>
+          <h2>Main Locations</h2>
+          <div class="cta-widget" data-widget-type="basic-facility-locations-list" data-facilities="{{ mainFacilities.entities | widgetFacilitiesList | escape }}"></div>
+          <a class="usa-button usa-button-secondary" href="{{ entityUrl.path }}/locations">
+            See all locations <i class="fa fa-chevron-right">&nbsp;</i>
+          </a>
+        </section>
+
+        <!-- "Manage your health online" section -->
+        <section>
+          <h3>Manage your health online</h3>
+          <div class="usa-grid usa-grid-full">
+            <div class="usa-width-one-half facility-hub-cta-item">
+              <a href="https://www.va.gov/health-care/refill-track-prescriptions/">
```

Comment:
The hover state for these links looks better if you style these links as `display: block`.